### DISCUSSION TITLE
Random number variable

### DISCUSSION THREAD

**Discussion by JXYZP-G** [gameserverapp/Platform#361] [Suggestions]
This was a post on the wrong location so here is a repost :)

Original Post:
> I'm not completely sure if this belongs here. but it has to do with shop it has to do with the variables of ingame command(ark) for 
> example.
> 
> GSA Command: `giveitem {gameid} "Blueprint'/Game/PrimalEarth/CoreBlueprints/Items/Armor/Metal/PrimalItemArmor_MetalHelmet.PrimalItemArmor_MetalHelmet'" 1 1 5`
> 
> I think there should be a option to randomize the quality number(5) in this case. to {quality) and somewhere to configure this that it stays between 5-7.
> 
> While writing this Something with custom variables came up in my head. Since maybe you want another quality each item. Its hard to explain i hope you understand something 😂  
> 
> edit:
> 
> Something like supply crates:
> ![image](https://user-images.githubusercontent.com/43095469/222914605-8f5a1a50-e146-4a5b-80c0-3d81e25f5b97.png)
> 
> Or add this as functionality to the randomizer option.
>

**Comment by mvaessen** [comment]
This seems similar to an existing suggestion: #282
